# Exception
#raise Exception('myException')
#raise TypeError('typeError')